packages/styles/scss/themes/expressive/components/_ui-shell.scss

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/layout/convert';
+
+//-----------------------------
+// Accordion (expressive)
+//-----------------------------
+
+/// Accordion styles (expressive)
+/// @access private
+/// @group accordion-expressive
+@mixin ui-shell-expressive {
+  //$side-nav-link-padding: calc(
+  //  (#{$container-02} - #{map-get($body-long-01, 'line-height')}) / 2
+  //);
+  //$switcher-nav-link-padding: calc(
+  //  (#{$container-02} - #{map-get($productive-heading-01, 'line-height')}) / 2
+  //);
```

Comment:
```suggestion
```

I think these lines are meant to be removed? (If the flex box approach sure works)

Reply:
yeah forgot to remove them 😅 thanks!